### PR TITLE
feat: better error message on wrong os-release file

### LINKED ISSUE #499: Better error message when os-release file parsing goes wrong
image

## Additional Details
OS: Artix Linux x86_64
Kernel: 6.4.1-zen1-1-zen
Installation: `cargo install topgrade`
Version: 12.0.0, also newest f3b1d2dfb3b38d73b14d7254a76c33359d062370 tested

### PR DESCRIPTION
**Pull request by 6543**
closes  #499

- bump rust-ini get rid of the panic
- detect if something is wrong with os-release

## Standards checklist:

- [x] The PR title is descriptive.
- [x] I have read `CONTRIBUTING.md`
- [x] The code compiles (`cargo build`)
- [x] The code passes rustfmt (`cargo fmt`)
- [x] The code passes clippy (`cargo clippy`)
- [x] The code passes tests (`cargo test`)

### CODE REVIEW

File: src/error.rs [hunk 1]
```diff
@@ -14,6 +14,10 @@ pub enum TopgradeError {
     #[cfg(target_os = "linux")]
     UnknownLinuxDistribution,
 
+    #[error("File \"/etc/os-release\" do not exist or is empty")]
```
Comment (SteveLauC) [inline src/error.rs:17]: #[error("File \"/etc/os-release\" does not exist or is empty")]

**Review by SteveLauC — CHANGES_REQUESTED** [review]

**Comment by SteveLauC** [comment]
Except for that little grammar mistake, this PR LGTM, thanks! :)

File: src/error.rs (hunk 1 above)
Comment (6543) [inline src/error.rs:17]:
```suggestion
    #[error("File \"/etc/os-release\" does not exist or is empty")]
```

**Comment by 6543** [comment]
Done :)

As pulls are squash merged i did just append the commit.
If i have to swuash them tell me

@SteveLauC 🚀

**Review by SteveLauC — APPROVED** [review]

**Pull request merged**